src/hooks/analytics/analytics.ts

```diff
@@ -29,7 +28,7 @@ export const hook = async (options: { command: string, flags: any, config: IConf
 
     // Prompt question if user allow chectl to collect data anonymous data.
     if (!options.flags.telemetry && !segmentTelemetry) {
-      const confirmed = await cli.confirm(`${getProjectName()} would like to collect data about how users use cli commands. Participation is voluntary and when you choose to participate chectl automatically sends statistic usage about how you use the cli [y/n]:`)
+      const confirmed = await cli.confirm('Enable CLI usage data to be sent to Red Hat online services. [y/n]:')
```

Comment:
```suggestion
      const confirmed = await cli.confirm('Enable CLI usage data to be sent to Red Hat online services. [y/n]')
```
`:` is added automatically